db/migrate/20200911140139_create_test_groups.rb

```diff
@@ -0,0 +1,18 @@
+class CreateTestGroups < ActiveRecord::Migration[5.2]
+  def change
+    create_table :test_groups do |t|
+      t.string :name, null: false
+      t.references :company, index: true, foreign_key: true
+
+      t.timestamps
+    end
+
+    add_reference :tests, :test_group, index: true, foreign_key: true
```

Comment:
Columns for tests
Add please value int column, by default it's 0
Add please index column, by default it's 1
Add description column as text
add uniq indexes for DB on test_group and name | name and index

Reply:
"index" to "module"